### PR TITLE
Cas 8731  	Implemented MSMetaData::getSpwToScansMap()

### PR DESCRIPTION
**Pull request by dmehring**
Also  	Place test for getScansForSpw() in its own block in tMSMetaData.

### CODE REVIEW

File: ms/MSOper/MSMetaData.cc [hunk 1]
```diff
@@ -1428,6 +1428,13 @@ std::map<ScanKey, std::set<uInt> > MSMetaData::getScanToSpwsMap() const {
     return scanToSpwMap;
 }
 
+vector<std::set<ScanKey> > MSMetaData::getSpwToScansMap() const {
```
Comment (juliantaylor) [inline ms/MSOper/MSMetaData.cc]: spw's here are always indexed 0 to vector.size()?
I would have expected a map int -> set (or multi_map)

Comment (dmehring) [inline ms/MSOper/MSMetaData.cc]: Yes. Values of spw are simply the row numbers in the SPECTRAL_WINDOW subtable, so a vector seemed like the most natural choice.

Comment (juliantaylor) [inline ms/MSOper/MSMetaData.cc]: ok assuming this is consistent in the api
thanks

**Comment by dmehring** [comment]
Thanks.

**Pull request merged**